Elias/exercise-39.js

```diff
@@ -0,0 +1,49 @@
+// 1. 2D Array
+
+let board = [
+    [1, 2, 3],
+    ["quick", "brown", "fox", "jumped", "over", "lazy", "dog"],
+    [true, false]
+];
+const newArray = []
+const loopTwoDarray = (array) => {
+array.map((item) => {
+        return item.map((j, index) => {
+            newArray.push(j)
+        })
+    })
+}
+
+loopTwoDarray(board)
+
+console.log(newArray.join(' '))
+
+// 2. Doggo
+
+const doggo = {
+    name: "Bodo",
+    breed: "German Sheaperd",
+    favoritFood: [
+        "sheep", "chocolate", "chicken"
+    ],
+}
+
+const secondElement = doggo.favoritFood[1];
+console.log(secondElement);
+console.log(doggo.favoritFood.map((val) => val).join(' '));
+
+// 3. 
+const recipes = {
+    ingredients: {
+        butter: "200gram",
+        sugar: "500gram",
+        flour: "1kg",
+        eggs: "3 large" 
+    }
```

Comment:
// 3.5 ==> You could make a method within recipes object and store the value of each ingredient in the ingredients object.
Like for example you can declare the method after ingredients object ==>
printIngredients() {
    return Object.values(this.ingredients);
  }

then you can just print the values ==>
console.log("print ==>",recipes.printIngredients());